### PR TITLE
Choose best image in embedly schema #140

### PR DESCRIPTION
**Pull request by sarracini**
@jaredkerim r?

### CODE REVIEW

File: embedly/embedly/schema.py [hunk 1]
```diff
@@ -49,4 +49,13 @@ def get_domain(url):
             if get_domain(image.get('url', '')) not in disallowed_domains
         ]
 
+        def cmp_images(img_a, img_b):
+            return cmp(
+                img_b['width'] * img_b['height'],
+                img_a['width'] * img_a['height'],
+            )
```
Comment (pdehaan) [inline embedly/embedly/schema.py]: And this would still work if there was only one image, correct?

Comment (sarracini) [inline embedly/embedly/schema.py]: yup!

Comment (jaredkerim) [inline embedly/embedly/schema.py]: Python is ✨ magic ✨

**Comment by jaredkerim** [comment]
r+!

**Pull request merged**